### PR TITLE
Clarify dockerfile and path inputs

### PR DESCRIPTION
**Pull request by zappy-shu**
Clarify the documentation for the dockerfile and path action inputs including how they relate to each other.

Fixes https://github.com/docker/build-push-action/issues/11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,11 +86,13 @@ Example:
...
 
-Path to run the docker build from. Defaults to `.`.
+Path to the build context. Defaults to `.`
 
 ### `dockerfile`
 
-Name of the Dockerfile. Defaults to `{path}/Dockerfile`.
+Path to the Dockerfile. Defaults to `{path}/Dockerfile`
+
+Note when set this path is **not** relative to the `path` input but is instead relative to the current working directory.
```
Comment (justincormack) [inline README.md]: This is really confusing. If the `path` is `.` then it is relative to the `path`. What do you mean exactly?

Comment (justincormack) [reply README.md]: Do you mean "if you change the default it is relative to the current working directory not the `path`"?

Comment (zappy-shu) [reply README.md]: Yeah pretty much. 

```
path: path/to/wherever
dockerfile: my/Dockerfile
```

Is equivalent to `docker --file my/Dockerfile path/to/wherever` which would mean your directory structure looked something like:

```
.
- my/
  - Dockerfile
- path/
  - to/
     - wherever/
```

"Path to the Dockerfile. Defaults to `{path}/Dockerfile`" Is pretty much a copy/paste of the `docker help build` output:

```
  -f, --file string             Name of the Dockerfile (Default is
                                'PATH/Dockerfile')
```

Happy to change to something more clear if you have better wording

Comment (justincormack) [reply README.md]: I think its ok for now, I think however you explain it it might be confusing...

**Pull request merged**